src/DynamoCoreWpf/ViewModels/Core/WorkspaceViewModel.cs

```diff
@@ -221,14 +227,20 @@ public DynamoPreferencesData DynamoPreferences
         /// of Graph.Json.
         /// </summary>
         [JsonProperty("Camera")]
-        public CameraData Camera => DynamoViewModel.BackgroundPreviewViewModel.GetCameraInformation() ?? new CameraData();
+        public CameraData Camera => DynamoViewModel.BackgroundPreviewViewModel?.GetCameraInformation() ?? new CameraData();
 
         /// <summary>
         /// ViewModel that is used in InCanvasSearch in context menu and called by Shift+DoubleClick.
         /// </summary>
         [JsonIgnore]
         public SearchViewModel InCanvasSearchViewModel { get; private set; }
 
+        /// <summary>
+        /// ViewModel that is used in NodeAutoComplete feature in context menu and called by Shift+DoubleClick.
+        /// </summary>
+        [JsonIgnore]
+        public SearchViewModel NodePortAutoCompleteSearchViewModel { get; private set; }
```

Comment:
How about simplifying this name to `AutoCompleteSearchViewModel`? The long name is quite confusing IMO.

Reply:
How about `NodeAutoCompleteSearchViewModel`, AutoComplete itself is not specific enough IMO